packages/core/useConfirmDialog/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Ref } from 'vue-demi'
+import { createEventHook, EventHook, EventHookOn, Fn } from '@vueuse/core'
+
+export interface useConfirmDialogReturn {
+  /*
+  * Opens the dialog
+  */
+  showDialog: Fn
+
+  /**
+   * Confirms and closes the dialog. Triggers a callback inside `onConfirm` hook.
+   * Can accept any data and to pass it to `onConfirm`.
+   */
+  confirm: (data?: any) => void
```

Comment:
Is it somehow possible to get this generic typed? (same for `cancel`)

Reply:
Let's see what I can do